command/issues/list/list_test.go

```diff
@@ -238,3 +110,28 @@ func TestListSARIF(t *testing.T) {
 		}
 	})
 }
+
+func TestGetIssuesData(t *testing.T) {
```

Comment:
Looks good. Try adding one more test case with a combination of file and directory as well?

Reply:
Wanted to do that, but `filterIssuesByPath` only takes a single argument (path).

<sup>I forgot to change the test function name</sup>